src/extension/package.json

```diff
@@ -62,6 +62,10 @@
           "type": "boolean",
           "default": false,
           "description": "Enable upcoming features; Warning: may cause stability issues."
+        },
+        "wts.legalResources": {
+          "type": "null",
+          "markdownDescription": "[Privacy Link](https://privacy.microsoft.com/en-US/privacystatement) | [Terms of Use](https://www.microsoft.com/en-us/legal/intellectualproperty/copyright/default.aspx)"
```

Comment:
This seems like a weird place to put such info. Almost as if you were hiding it haha. Why not put it on the wizard or something?

Reply:
This was a decision made by Stephane, since this is an open-source project, people can use our product as they please and the best way to handle this was to put it in settings to have a balance between the nature of the product and always having it there.

Reply:
Yea I agree, the settings was not really designed for this but from discussion with Stephane, we decided to put it here